src/common/snippets/src/lowered/linear_ir.cpp

```diff
@@ -201,12 +205,12 @@ void LinearIR::unregister_expression(const ExpressionPtr& expr) {
 }
 
 LinearIR::exprIt LinearIR::insert(constExprIt pos, container::value_type&& value) {
-    register_expression(value, m_config.m_manual_build_support);
+    register_expression(value, m_config.m_manual_build_support, get_inserted_expr_exec_num(pos));
```

Comment:
There is idea to automatically call `get_loop_manager()->reorder_loop_ports(expr->get_loop_ids());` in each `LinearIR::insert()` instead of call it in the corresponding helpers.
However, I'm not sure in this because for me `insert` methods should be maximally simple - just to insert expression to LIR. Other needed machinations should be completed by another entity :thinking: (`LinearIR` =, for example)